terraria-tmodloader/terraria-tmodloader.json

```diff
@@ -158,60 +166,80 @@
   },
   "install": [
     {
-      "files": [
-        "https://github.com/tModLoader/tModLoader/releases/download/v${tversion}/tModLoader.Linux.v${tversion}.tar.gz"
-      ],
-      "type": "download"
+      "target": "server",
+      "type": "mkdir"
+    },
+    {
+      "target": "${wp}",
+      "type": "mkdir"
     },
     {
+      "target": "${mp}",
+      "type": "mkdir"
+    },
+    {
+      "type": "command",
       "commands": [
-        "mv ./${mp}/enabled.json ./",
-        "rm -r ./server",
-        "mkdir ./server",
-        "tar -xf tModLoader.Linux.v${tversion}.tar.gz --directory ./server",
-        "chmod +x ./server/tModLoaderServer",
-        "chmod +x ./server/tModLoaderServer.bin.x86",
-        "chmod +x ./server/tModLoaderServer.bin.x86_64",
-        "mkdir ./${wp}",
-        "mkdir ./${mp}",
-        "rm tModLoader.Linux.v${tversion}.tar.gz"
-      ],
-      "type": "command"
+        "wget https://github.com/tModLoader/tModLoader/releases/download/${version}/tModLoader.zip"
```

Comment:
Use the download operator over wget

Reply:
i wasn't able download with the download operator, that's why I am using wget

Reply:
What happens with that? download should work the same as wget to a degree.

Reply:
I'm getting : 'Failed to install server', which is weird because when I made the template 5 months ago, it was working...

Reply:
seems to be working now for some reasons...